### ISSUE TITLE
[P2][WIP][Sourcing][Accessibility][Multi input list][more input]The selection list does not have a label.the selection list relates to the input field, and the values of the selection list are also not output because output is only made in forms mode

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10290]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The selection list does not have a label.
the selection list relates to the input field, and the values of the selection list are also not output because output is only made in forms mode

#### If this is a bug, please provide steps for reproducing it.
The selection list does not have a label.
the selection list relates to the input field, and the values of the selection list are also not output because output is only made in forms mode
tested on multi-input mobile mode example

#### Is there anything else we should know?
reported by customer

**Comment by I543348** [comment]
ATL comment:
No hint for Screen reader users when focusing the input field with TAB key about the values in the field.

**Comment by droshev** [comment]
@I543348 Can you provide more detail information and some screenshots?

**Comment by droshev** [comment]
@I543348 The agreement is that you provide us with all the input so we can work on the issue.

**Comment by fkolar** [comment]
Updated to WIP until ticket has all the information.

**Comment by nikolay-kolarov** [comment]
Hello,

My assumption is that this basically duplicates #10293. With it, we will add a label of the tokenizer list and we will get rid of the second listbox role, which will improve the structure.
The reading in virtual mode is working as expected - it is the same in the official aria example :https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/ - if you navigate with virtual cursor, you will get only listbox (and its label) and if you navigate further in the listbox in forms mode then you get the values.

So if there is no way to get further detailed info for this issue from the customer, I propose to close it as duplicate of #10293.

Best Regards,
Nikolay

**Comment by droshev** [comment]
@I543348 The "mobile example" is not created to be tested on desktop. The main goal is to show how the mobile multi input looks and behave. You should not use the mobile example for JAWS testing

**Comment by I543348** [comment]
ATL Re-verification needed

**Comment by sagar7259** [comment]
Hello,

The selection list should be read to the users even in forms mode of JAWS with its right UI type. Here the information " x tokens present" or any format which conveyed the same information is also holds good. But the total number of tokens present should be read to the users.
Regards,
Sagar (ATL team)
